src/12_scopes.py

```diff
@@ -6,8 +6,9 @@
 
 def changeX():
     x = 99
+    return x
```

Comment:
Use the `global` keyword here Petar, this will allow the function to use the global variable x, see below..

```
def changeX():
    global x
    x = 99
```